app/src/main/kotlin/com/google/samples/apps/nowinandroid/ui/interests2pane/InterestsListDetailScreen.kt

```diff
@@ -103,15 +86,15 @@ internal fun InterestsListDetailScreen(
         detailPane = {
             NavHost(
                 navController = nestedNavController,
-                startDestination = TOPIC_ROUTE,
-                route = DETAIL_PANE_NAVHOST_ROUTE,
+                startDestination = TopicPlaceholderDestination::class,
```

Comment:
StartDestination can be a `KClass<T>` or an `object instance of T`, so this can be simplified to just `TopicPlaceholderDestination`.